httpapi/coreapi/types.go

```diff
@@ -19,6 +19,16 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 )
 
+// MonetaryValue defines user format to represent currency type
+// Value string representation of decimal number
+// ChainID hex bytes representing the chain where the currency is relevant
+// ID string representing the Currency (USD|ETH|0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2(DAI)...)
+type MonetaryValue struct {
+	Value   string             `json:"value" enums:"decimal"`
```

Comment:
`enums` represents the Value to bee always `decimal` as a string. 
```suggestion
	Value   *documents.Decimal             `json:"value" swaggertype:"primitive,string"`
```